pages/apim/3.x/event-native-apim/event-native-apim-documentation.adoc

```diff
@@ -0,0 +1,145 @@
+[[event-native-apim-documentation]]
+= Documentation
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/event_native_apim_documentation.html
+:page-folder: apim/event-native-apim
+:page-layout: apim3x
+
+[label label-version]#New in version 3.20.0#
+[label label-version]#BETA release#
+
+== Overview
+
+_This page is for the draft event-native APIM docmentation that Andrew is writing for 3.20. It is here only for initial draft and review purposes. It will be split into different pages and (probably) moved to a new location later._
+
+== Asynchronous APIs
+
+In this release, you can use the gateway to connect to asynchrous APIs. You can configure asynchronous API connections using the RESTful interface to the management API.
```

Comment:
asynchrous -> asynchronous?

Reply:
That's the deliberate typo to see if you are paying attention.... ;)